Library/Formula/libhttpserver.rb

```diff
@@ -0,0 +1,41 @@
+require "formula"
+
+class Libhttpserver < Formula
+  homepage "https://github.com/etr/libhttpserver"
+  url "https://github.com/etr/libhttpserver/archive/v0.7.1.tar.gz"
+  head "https://github.com/etr/libhttpserver.git"
+  sha1 "b36c8a5f523c04613d6a36c86bbb31a2b92aab83"
+
+  option "enable-cpp11", "Enable c++11 std classes"
+  depends_on "libmicrohttpd"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+
+  option :universal
+
+  def install
+    ENV.universal_binary if build.universal?
+
+    ENV["LIBMICROHTTPD_CFLAGS"] = '$(pkg-config libmicrohttpd --cflags)'
+    ENV["LIBMICROHTTPD_LIBS"] = '$(pkg-config libmicrohttpd --libs)'
+    args = [
+      "--disable-dependency-tracking",
+      "--disable-silent-rules",
+      "--prefix=#{prefix}"
+    ]
+
+    args << "--enable-cpp11" if build.include? "enable-cpp11"
+
+    inreplace 'bootstrap', 'libtoolize', 'glibtoolize'
+    system "./bootstrap"
+    mkdir "build"
+    cd "build" do
+        system "../configure", *args
+        system "make", "install"
```

Comment:
https://github.com/etr/libhttpserver/blob/master/README.md
The readme says we must add the following option to build in the same directory, which is heavily discouraged. So a make it in the build directory.
> --enable-same-directory-build: enable to compile in the same directory. This is heavily discouraged.